docs/user/integration/dynatrace/README.md

```diff
@@ -211,13 +211,13 @@ Depending on they way your applications emit metrics, choose one of the followin
             output:
                 otlp:
                     endpoint:
-                        value: http://dynatrace-exporter-opentelemetry-collector.${DYNATRACE_NS}:4317
+                        value: http://dynakube-telemetry-ingest.${DYNATRACE_NS}:4317
         EOF
         ```
 
-- If your application pushes metrics in delta temporality with OTLP, you can use a MetricPipeline to push metrics directly.
+- If your application pushes metrics in delta temporality with OTLP (and you don't want to collect metrics using the MetricPipeline `istio` and `prometheus` input), push the metrics directly to Dynatrace. This setup will ship data directly to the Dynatrace backend and will avoid unneccessary processing in additional components.
 
-  To use this setup, you must explicitly enable the "delta" aggregation temporality in your applications. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temperability.
+  To use this setup, you must explicitly enable the "delta" aggregation temporality as prefered tempoarility in your applications. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temporability.
```

Comment:
```suggestion
  To use this setup, you must explicitly enable the "delta" aggregation temporality as preferred temporality in your applications. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temporality.
```